frontend/components/Landing/MeetupIndicator.vue

```diff
@@ -3,33 +3,51 @@
     </div>
-    <div class="relative mx-auto max-w-7xl px-4 py-12 sm:px-6 lg:px-8 lg:py-16">
+    <div class="mx-auto max-w-7xl px-4 py-12 sm:px-6 lg:px-8 lg:py-16">
       <div class="md:ml-auto md:w-1/2 md:pl-10">
-        <h2 class="uppercase font-semibold text-zinc-300">
+        <h2 class="ppercase font-semibold text-2xl text-zinc-300">
           Następny meetup
-          <span class="text-laravel">&</span>
-          za miesiąc
+          <span class="text-laravel">już</span>
+          za miesiąc 
         </h2>
-        <div class="mt-3 text-3xl font-bold tracking-tight text-white sm:text-4xl">
-          Laravel Poland Meetup #22
+        <div class="mt-3 text-3xl font-bold tracking-tight text-white sm:text-4xl"  v-if="data.name" >
+          {{data.name}}
         </div>
-        <p class="mt-3 text-lg text-zinc-300">
-          Spotykamy się już 22 listopada lorem ipsum dolor sit amet, consectetur adipisicing elit. Blanditiis delectus dolorem eius placeat quaerat quia quisquam. Ab ad assumenda blanditiis et eveniet ex iure, laboriosam nostrum porro quam! Error, id.
+        <p class="mt-3 text-2xl text-zinc-300 leading-8">
+          <strong class="text-laravel">{{date}}</strong><br>
+          Zapraszamy fascynatów Laravela i nie tylko!<br>
+          Wstęp free.<br>
+          <span v-if="data.location === 'online'">Tym razem widzimy się online!</span>
+          <span v-else>Miejsce spotkania: {{data.location}}!</span>
         </p>
         <div class="mt-8">
-          <div class="inline-flex gap-4 rounded-md shadow">
-            <a href="#" class="inline-flex items-center justify-center rounded-md border border-transparent bg-white px-5 py-3 text-base font-medium text-zinc-900 hover:bg-zinc-200">
-              Zobacz agendę i szczegóły
-            </a>
-            <a href="#" class="inline-flex items-center justify-center rounded-md border border-transparent bg-zinc-500 px-5 py-3 text-base font-medium text-zinc-900 hover:bg-zinc-600">
-              Zobacz poprzednie meetupy
-            </a>
+          <div class="block gap-4 tracking-wide rounded-md shadow sm:block md:block lg:w-3/5">
+            <a href="https://www.subscribepage.com/lpm23?fbclid=IwAR2mdIHtj7U2kCAlRMzpzbenpyKdO6HkdMsXTeFjzBm1lypc5oOwmsIfSzI" class="flex items-center justify-center rounded-md border border-transparent bg-laravel px-8 py-3 my-3 text-xl font-bold text-white hover:bg-red-800 hover:scale-105 duration-300 ">Zapisz się!</a>
+            <router-link :to="{ name: 'meetups' }" class="flex items-center justify-center rounded-md border border-transparent bg-zinc-100 px-5 py-3 text-base font-medium text-zinc-900 hover:bg-zinc-300 hover:scale-105 duration-300">Poprzednie meetupy</router-link>
           </div>
         </div>
       </div>
     </div>
   </div>
 </template>
+
+<script>
+import DataMeetup from '../../../resources/meetups/2022-11-10-laravel-poland-meetup-23.json'
```

Comment:
I think it's already configured. @paulinaszkudlarek, you just need to replace this `../../../` with `@/`.

Reply:
You are right, it is, but then we can use:
`@/../resources/meetups/xyz.img`
or change alias to point `/` instead of `/frontend` - but this probably will require more changes in other files.

In the project dir there are `resources` and `fronted` folders on the same level. At this moment alias is pointing to `/fronted`.
https://github.com/blumilksoftware/laravel-poland-meetup/blob/main/vite.config.js#L7